interp/type.go

```diff
@@ -1183,6 +1187,26 @@ func (t *itype) id() (res string) {
 	return res
 }
 
+func constTypeString(t reflect.Type) (s string) {
+	cv, ok := reflect.New(t).Elem().Interface().(constant.Value)
+	if !ok {
+		return
+	}
```

Comment:
we're returning the empty string here, but in the caller we do not check whether res is empty, so I think this is going to end up being silently ignored, won't it?
So are you sure you don't want to panic instead?

Reply:
Returning an empty string if unresolved is correct for use in `id()`. We do not want to panic here, as it would avoid us to return a proper error during parse.